ivy/functional/frontends/torch/tensor_functions.py

```diff
@@ -14,8 +15,8 @@ def is_tensor(obj):
 # def is_storage(obj):
 # 	return ivy.is_storage(obj)
 
-# def is_complex(obj):
-# 	return ivy.is_complex(obj)
+def is_complex(obj):
+ 	return Tensor(ivy.is_complex(obj))
```

Comment:
Ivy doesn't have `is_complex` in its api. You would need to make an implementation here using some of the Ivy Functions currently implemented